packages/notifi-react-card/lib/components/intercom/ChatMessageSection.tsx

```diff
@@ -1,40 +1,96 @@
-import clsx from 'clsx';
 import React from 'react';
+import { Virtuoso } from 'react-virtuoso';
 
-import { ChatMessageDate, ChatMessageDateProps } from './ChatMessageData';
+import { useIntercomChat } from '../../hooks/useIntercomChat';
+import { ChatMessageDate, ChatMessageDateProps } from './ChatMessageDate';
+import {
+  ChatWindowIntroSection,
+  ChatWindowIntroSectionProps,
+} from './ChatWindowIntroSection';
 import { MessageList, MessageListProps } from './MessageList';
 
 export type ChatMessageSectionProps = Readonly<{
   classNames?: Readonly<{
-    container: string;
-    content: string;
-    date: ChatMessageDateProps['classNames'];
+    chatWindowIntro: ChatWindowIntroSectionProps['classNames'];
     messageList: MessageListProps['classNames'];
+    date: ChatMessageDateProps['classNames'];
   }>;
   chatMessageSectionIntroContent?: string;
 }>;
 
 export const ChatMessageSection: React.FC<ChatMessageSectionProps> = ({
   classNames,
-  chatMessageSectionIntroContent = 'What can we help you with today?',
+  chatMessageSectionIntroContent,
 }) => {
+  const {
+    conversation,
+    setIsScrolling,
+    setVisibleRange,
+    setAtTop,
+    isLoading,
+    hasNextPage,
+  } = useIntercomChat({
+    conversationId: '8c5ccf41d93c4bb9b5e1e1014002e923',
+  });
+
   return (
-    <div
-      className={clsx(
-        'NotifiIntercomChatMessageSection__container',
-        classNames?.container,
+    <>
+      {conversation.feed.length === 0 ? (
```

Comment:
Could you possibly do `conversation.feed.length === 0 ? <ChatWindowIntroSection /> : <Virtuoso />`?

I am just thinking about readability.